--- bijiben-40.1/data/appdata/org.gnome.Notes.appdata.xml.in-orig	2022-04-23 21:48:27.405376036 -0400
+++ bijiben-40.1/data/appdata/org.gnome.Notes.appdata.xml.in	2022-04-23 21:49:03.371708065 -0400
@@ -22,19 +22,19 @@
   </kudos>
   <screenshots>
     <screenshot type="default">
-      <image width="800" height="647">https://gitlab.gnome.org/GNOME/gnome-notes/raw/master/data/appdata/default.png</image>
+      <image>https://gitlab.gnome.org/GNOME/gnome-notes/raw/master/data/appdata/default.png</image>
     </screenshot>
     <screenshot>
       <caption>Edit view</caption>
-      <image width="800" height="647">https://gitlab.gnome.org/GNOME/gnome-notes/raw/master/data/appdata/edit.png</image>
+      <image>https://gitlab.gnome.org/GNOME/gnome-notes/raw/master/data/appdata/edit.png</image>
     </screenshot>
     <screenshot>
       <caption>Select view</caption>
-      <image width="800" height="647">https://gitlab.gnome.org/GNOME/gnome-notes/raw/master/data/appdata/select.png</image>
+      <image>https://gitlab.gnome.org/GNOME/gnome-notes/raw/master/data/appdata/select.png</image>
     </screenshot>
     <screenshot>
       <caption>List view</caption>
-      <image width="800" height="647">https://gitlab.gnome.org/GNOME/gnome-notes/raw/master/data/appdata/list.png</image>
+      <image>https://gitlab.gnome.org/GNOME/gnome-notes/raw/master/data/appdata/list.png</image>
     </screenshot>
   </screenshots>
 
